src/input.rs

```diff
@@ -52,6 +52,8 @@ pub const KEY_BACKWARD: u16 = 193;
 pub const KEY_FORWARD: u16 = 194;
 pub const KEY_ROTATE_DISPLAY: u16 = 153;
 pub const SLEEP_COVER: u16 = 59;
+// INVERT_BUTTON_SCHEME is not a real device key code and is used to support software toggles within this design
+pub const INVERT_BUTTON_SCHEME: u16 = 65535;
```

Comment:
Since this is a fake key value similar to `KEY_ROTATE_DISPLAY`, you might want to name it `KEY_BUTTON_SCHEME`. And set the value of `KEY_ROTATE_DISPLAY` to `0xfffe`?